.travis.yml

```diff
@@ -45,20 +45,13 @@ matrix:
     - php: '7.1'
```

Comment:
Please create a dev-kit PR for this, otherwise it gets overridden

Reply:
Done